qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -157,6 +158,13 @@ def pass_manager(self, pass_manager_config, optimization_level=None) -> PassMana
             if pass_manager_config.routing_method != "none":
                 init.append(ElidePermutations())
             init.append(RemoveDiagonalGatesBeforeMeasure())
+            # Target not set on RemoveIdentityEquivalent because we haven't applied a Layout
```

Comment:
I think I would expect the transpiler to remove `Ry(0)` & co even as a very basic optimization, should we also include this target-free run in optimization level 1?

Reply:
My concern with doing it for level 1 was more about small angles like `Ry(1e-8)` will be removed and I was worried that might be surprising to people especially at level 1. But I don't feel super strongly about it, it's easy enough to add it to level 1 if you think that's better.

Reply:
Hm you're right, I didn't think that these angles are already removed already... as rule we should maybe not increase the level of approximation that is already happening (e.g. cutoffs that unitary re-synthesis does). On level 1 we don't do re-synthesis, right? So then I agree we shouldn't add it one 1 yet 🙂

Reply:
Level 1 will run 1q re-synthesis, but via `Optikize1qGatesDecomposition` not `UnitarySynthesis`. I can't remember how that pass handles tolerances, I'll have to check.